### PR TITLE
Polish translation 3.5.0

### PR DESCRIPTION
**Pull request by theYoursTruly**
### When merged this pull request will:

1. Add missing Polish translations
2. Change text _MK6_ to _Mk6_ in mk6mortar module
3. Remove duplicate german translation of _STR_ACE_finger_indicatorColor_description_ in finger module

### CODE REVIEW

**Comment by esteldunedain** [comment]
:+1:

File: addons/medical/stringtable.xml [hunk 1]
```diff
@@ -957,12 +959,15 @@
         </Key>
         <Key ID="STR_ACE_Medical_Adenosine_Display">
             <English>Adenosine autoinjector</English>
+            <Polish>Autostrzykawka adenozyny</Polish>
         </Key>
         <Key ID="STR_ACE_Medical_Adenosine_Desc_Short">
             <English>Used to counter effects of Epinephrine</English>
+            <Polish>Stosowana do odwracania efektu działania adrenaliny.</Polish>
```
Comment (gienkov) [inline addons/medical/stringtable.xml]: "zwalczania efektu" could be better

File: addons/medical/stringtable.xml [hunk 2]
```diff
@@ -957,12 +959,15 @@
...
         </Key>
         <Key ID="STR_ACE_Medical_Adenosine_Desc_Use">
             <English>A drug used to counter the effects of Epinephrine</English>
+            <Polish>Lek stosowany do odwracania efektu działania adrenaliny.</Polish>
```
Comment (gienkov) [inline addons/medical/stringtable.xml]: same as above

**Comment by bux578** [comment]
@theYoursTruly Could you fix the merge conflicts, please?

File: addons/medical/stringtable.xml [hunk 3]
```diff
@@ -957,12 +959,15 @@
         </Key>
         <Key ID="STR_ACE_Medical_Adenosine_Display">
             <English>Adenosine autoinjector</English>
+            <Polish>Autostrzykawka adenozyny</Polish>
```
Comment (gienkov) [inline addons/medical/stringtable.xml]: Autostrzykawka z adenozyną

File: addons/medical/stringtable.xml [hunk 4]
```diff
@@ -2871,11 +2877,13 @@
             <English>Locations boost training</English>
             <Czech>Místa pro vylepšení zkušeností</Czech>
             <Italian>Località aumentano addestramento</Italian>
+            <Polish>Lokacje zwiększają poziom medyczny</Polish>
```
Comment (gienkov) [inline addons/medical/stringtable.xml]: In master already

File: addons/medical/stringtable.xml [hunk 5]
```diff
@@ -2871,11 +2877,13 @@
...
         </Key>
         <Key ID="STR_ACE_Medical_MedicalSettings_increaseTrainingInLocations_Description">
             <English>Boost medic rating in medical vehicles or near medical facilities [untrained becomes medic, medic becomes doctor]</English>
             <Czech>Zlepšit zkušenosti zdravotníka v medickém vozidle nebo poblíž zdravotního zařízení [nezkušení se stane zdravotníkem, zdravotník se stane doktorem] </Czech>
             <Italian>Aumenta il rating medico in veicoli medici o vicino strutture mediche [non addestrato diventa medico, medico diventa dottore]</Italian>
+            <Polish>Zwiększenie poziomu medyków w pojazdach medycznych lub przy budynkach medycznych [niedoświadczony zostaje medykiem, medyk zostaje doktorem]</Polish>
```
Comment (gienkov) [inline addons/medical/stringtable.xml]: In master already

File: addons/medical/stringtable.xml (hunk 1 above)
Comment (theYoursTruly) [inline addons/medical/stringtable.xml]: true

**Comment by gienkov** [comment]
I've made another PR with **all** missing translations (this Pr is missing some still and has conflicts)
https://github.com/acemod/ACE3/pull/3421

**Comment by theYoursTruly** [comment]
Should I stop fixing this one then?

**Comment by gienkov** [comment]
You would have to do yours from scratch since you are working on master, you should always do new branches if you want to make PR's.

**Comment by theYoursTruly** [comment]
Alright then.

**Comment by jonpas** [comment]
Replaced by #3421 thanks anyways @theYoursTruly

**Pull request closed without merging**